samples/sensor/bmp085/src/main.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024, Mario Paja
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(main, 0);
+
+#include <zephyr/kernel.h>
+
+#include <zephyr/sys/printk.h>
+#include <zephyr/sys_clock.h>
+#include <stdio.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/i2c.h>
+
+#if !DT_HAS_COMPAT_STATUS_OKAY(bosch_bmp085)
+#error "No bosch, bmp085 compatible node found in the device tree"
+#endif
+
+int main(void)
+{
+	const struct device *const bmp085 = DEVICE_DT_GET_ANY(bosch_bmp085);
```

Comment:
We are trying to avoid introducing new driver-specific sensor samples. If you would like to introduce a new sample, please make it a generic pressure sensor sample, similar to `samples/sensor/accel_polling/` and `samples/sensor/magn_polling/`